Fix: bookmark_drag_drop_in_library and paste_bookmark_in_library

ranjit-exalt opened the pull request:
Applying this patch will fix the following test cases:
bookmark_drag_drop_in_library #4549
paste_bookmark_in_library #4550

Commit:
Applying this patch will fix following test cases:
bookmark_drag_drop_in_library.py
paste_bookmark_in_library.py

spronoza reviewed (changes requested):
Failed on Linux and OSX
![image](https://user-images.githubusercontent.com/46671906/72993259-c3401b00-3dfd-11ea-8fbc-58d1164a0b27.png)

Commit:
Change log for "paste_bookmark_in_library":
1. Added missing image "other_bookmarks_option_pattern.png" for OSX, Linux and Win 10.

ranjit-exalt commented:
> Failed on Linux and OSX
> ![image](https://user-images.githubusercontent.com/46671906/72993259-c3401b00-3dfd-11ea-8fbc-58d1164a0b27.png)

Issue fixed. plz review.

tracywalker commented:
Both tests pass Win, Linux and OSX
